addon/bootstrap.js

```diff
@@ -42,7 +47,7 @@ XPCOMUtils.defineLazyModuleGetter(this, "Feature", `resource://${BASE}/lib/Featu
 
 async function startup(addonData, reason) {
   // `addonData`: Array [ "id", "version", "installPath", "resourceURI", "instanceID", "webExtension" ]  bootstrap.js:48
-  console.log("startup", REASONS[reason] || reason);
+  log.info("startup", REASONS[reason] || reason);
```

Comment:
How is `log.info` better than/different from `console.log`?

Reply:
I changed these to log.debug, which is kind of the point.  It's pref controllable now, so the bootstrap logs can actually be silenced.